doc/en/components/grids/_shared/sorting.md

```diff
@@ -0,0 +1,365 @@
+```
+
+And then with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) we can easily retrieve color from the palette.
+
+```scss
+$custom-theme: grid-theme(
+    $sorted-header-icon-color: color($custom-palette, "secondary", 500),
+    $sortable-header-icon-hover-color: color($custom-palette, "primary", 500)
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please refer to [`Palettes`](../themes/sass/palettes.md) topic for detailed guidance on how to use them.
+
+### Using Schemas
+
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/sass/schemas.md). A **schema** is a recipe of a theme.
+
+Extend one of the two predefined schemas, that are provided for every component, in this case - [`_light-grid`]({environment:sassApiUrl}/index.html#variable-_light-grid):  
+
+```scss
+// Extending the light grid schema
+$custom-grid-schema: extend($_light-grid,
+    (
+        sorted-header-icon-color: (igx-color:('secondary', 500)),
+        sortable-header-icon-hover-color: (igx-color:('primary', 500))
+    )
+);
+```
+
+In order to apply our custom schema we have to **extend** one of the globals ([`light`]({environment:sassApiUrl}/index.html#variable-light-schema) or [`dark`]({environment:sassApiUrl}/index.html#variable-dark-schema)), which is basically pointing out the components with a custom schema, and after that add it to the respective component themes:
+
+```scss
+// Extending the global light-schema
+$my-custom-schema: extend($light-schema, 
+    (
+        igx-grid: $custom-grid-schema
+    )
+);
+
+// Defining our custom theme with the custom schema
+$custom-theme: grid-theme(
+  $palette: $custom-palette,
+  $schema: $my-custom-schema
+);
+```
+
+Don't forget to include the themes in the same way as it was demonstrated above.
+
+<!-- end: Angular -->
+
+Add Styling section for Blazor
+
+### Demo
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:550px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-sorting-styling"
+           github-src="{ComponentSample}/sorting-styling" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:550px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-sorting-styling"
+           github-src="{ComponentSample}/sorting-styling" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-sorting-styling"
+           github-src="{ComponentSample}/sorting-styling" >
+
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.